### ISSUE TITLE
Bot goes offline without errors

### ISSUE THREAD

**Issue by f1oppa** [discordjs/discord.js#8172]
### Which package is this bug report for?

discord.js

### Issue description

I bought a VPS yesterday and uploaded my discord.js bot to it, Today morning I noticed that the bot is offline, but when i connected to my VPS via SSH. I saw, that there was no errors. I restarted the bot, and its online again.

### Code sample

```js
// the source code: https://github.com/f1oppa/toretto-open-source/
```

### Package version

v13.6.0

### Node.js version

v17.6.0

### Operating system

Ubuntu 22.04 LTS

### Priority this issue should have

High (immediate attention needed)

### Which partials do you have configured?

No Partials

### Which gateway intents are you subscribing to?

Guilds, GuildMembers, GuildMessages

**Comment by monbrey** [comment]
You've provided no information we can diagnose other than to say v13.6 is outdated and can crash due to text-in-voice channels.

Update to 13.8.1 and see if the issue persists.

**Comment by f1oppa** [comment]
On the VPS I have 13.8.1.

**Comment by Jiralite** [comment]
Frankly, there's no information here at all to diagnose the problem. Simply stating your bot goes offline tells us nothing. It might not even be discord.js related, as far as we know. Looking at your source code there seems to be no package.json too?

In other words, I don't find this to be a valid issue and there are no pointers to help us diagnose the issue. You need to provide a minimal reproducible code sample that reproduces this issue that we can use or start logging or debugging or doing any of the basic debugging techniques. I'll be closing this.

**Comment by f1oppa** [comment]
I'm sorry, that I couldn't explain my problem properly, so now I'll try to explain this now as best I can. 

So when I opened discord I saw that my bot is offline and didn't respond to any commands, but when I looked at the console it was like its running perfectly and there were no errors. 

I don't think, that I can send any code about this issue. 

And about the missing package.json file: It's there, but I forgot to upload to github. 

Anyways, if this happens again, I will send a video about this problem.
